ModulemdTranslationHelpers/Utils.py

```diff
@@ -11,6 +11,100 @@
     sys.exit(77)
 
 
+def get_latest_modules_in_tag(session, tag, debug=False):
```

Comment:
```suggestion
def get_latest_modules_in_tag(session, tag):
```

Drop the debug print statements in the code here and switch to using python's [logging](https://docs.python.org/3/library/logging.html) instead.